docs/guides/trace/dns.md

```diff
@@ -2,13 +2,13 @@
 title: 'Using trace dns'
 weight: 20
 description: >
-  Trace DNS requests.
+  Trace DNS queries and responses.
 ---
 
 ![Screencast of the trace dns gadget](dns.gif)
```

Comment:
I don't know. I think using 80 as the width is too small.

Reply:
I changed it to 80 for consistency in https://github.com/inspektor-gadget/inspektor-gadget/blob/89d68a4a6da56fefaa2637015b84b0b09958f73b/tools/demos/Makefile#L26-L30

I would like to have an option `kubectl gadget -o short` which looks nice on 80 columns by removing/reducing some fields...

Reply:
Sounds like responsive design on a CLI :) We could add something like `hide:sm` to hide columns by default when the screen width is smaller than a threshold value like, say, 80 - and that can be manually triggered by `-short` and `-wide`. I'm not too sure how much "magic" is welcome on a CLI, though...